docs/luigi-core-api.md

```diff
@@ -188,7 +190,9 @@ Luigi.readUserSettings();
 
 #### storeUserSettings
 
-Writes the user settings object to the **localStorage**. It is possible to choose a custom storage type by implementing the `settings.userSetting.storeUserSettings` function.
+Reads the user settings object.
```

Comment:
```suggestion
Writes the user settings object.
```
Unless I'm mistaken the description should be different than ` readUserSettings` above